packages/realm-web-integration-tests/harness/index.ts

```diff
@@ -40,16 +41,35 @@ const EXPECTED_ISSUES = [
     /was set with `SameSite=None` but without `Secure`/,
     // User / "refresh invalid access tokens" is posting an invalid token
     "Failed to load resource: the server responded with a status of 401 (Unauthorized)",
+    // Closing the watch streams, might yield this error (we're doing that three times)
+    "Failed to load resource: net::ERR_FAILED",
+    "Failed to load resource: net::ERR_FAILED",
+    "Failed to load resource: net::ERR_FAILED",
```

Comment:
Just curious, would the user have these thrown?

Reply:
Nope, they are not thrown - just logged to the console when the watch connection is closed and to be honest, I still don't really understand why (status code is 200 OK). I think it's perhaps the way we signal that we want to close (we're calling [`cancel`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader/cancel) on the response body stream), which should be ok:
> The cancel() method of the ReadableStreamDefaultReader interface cancels the stream, signaling a loss of interest in the stream by a consumer. The supplied reason argument will be given to the underlying source, which may or may not use it.